socialhome/streams/templates/streams/vue.html

```diff
@@ -0,0 +1,41 @@
+{% extends "streams/base.html" %}
+{% load staticfiles %}
+
+{% block content %}
+    {# Temporary hack; to be removed #}
+    <script>
+        window.templateData = {
+            contentList: [
+                {% for content in content_list %}
+                    {
+                        content: "{{ content.rendered|safe }}",
```

Comment:
What if `content.rendered` has a `"` in it, won't this break the object in the JS side?

Since this data is quite complex, might be better to make a custom template tag to do a `json.dumps(obj)` as here: https://stackoverflow.com/a/25538871/1489738 .. Also `name` should be done the same.